package.json

```diff
@@ -11,9 +11,11 @@
     "prebuild": "node scripts/prebuild.js",
     "prebuilds": "node scripts/prebuilds.js",
     "bundle": "webpack --display-modules",
+    "proto": "pbjs -t static-module -w commonjs -o proto/profile.js proto/profile.proto && pbts -o proto/profile.d.ts proto/profile.js",
```

Comment:
I know the files this is working with have `proto` as their extension, but `proto` along evokes memories of things like `__proto__`, which are unrelated. Can we just call this `protobuf`?